package.json

```diff
@@ -13,6 +13,7 @@
   "scripts": {
     "pretest": "yarn lint && yarn build",
     "test": "jest",
+    "test:dev": "jest --watch",
```

Comment:
simplify testing for developers, because running pretest can be annoying and will make the tests not start if I have a lint error like an unused import or whatever

Reply:
why not run `yarn test --watch`? :)

Reply:
@satya164 because I don't want to run lint/pretest, just the tests

Reply:
@slorber yea I was annoyed by that pretest too. maybe worth moving that to `yarn ci` or something